src/tools/pub/cmd/pub-uploader.md

```diff
@@ -31,6 +31,12 @@ They'll be added as an uploader after they accept the invitation.
 bob@example.com has been removed as an uploader for this package.
 ```
 
+{{site.alert.tip}}
+  Instead of specifying uploaders for each package you publish,
+  consider using a
+  [verified publisher](/tools/pub/verified-publishers).
```

Comment:
```suggestion
  Instead of specifying uploaders for each package that you publish,
  consider using a
  [verified publisher](/tools/pub/verified-publishers).
```